src/glossary.js

```diff
@@ -33,14 +32,14 @@ function forEach(values, callback) {
   return [].forEach.call(values, callback);
 }
 
-var itemTemplate = template(
-  '<li class="{{ glossaryItemClass }}">' +
-    '<button class="data-glossary-term {{ termClass }}">{{ term }}' +
-    '</button>' +
-    '<div class="{{ definitionClass }}">{{ definition }}</div>' +
-  '</li>',
-  {interpolate: /\{\{(.+?)\}\}/g}
-);
+var itemTemplate = function(values) {
+  return '<li class="' + values.glossaryItemClass + '">' +
```

Comment:
Looks fine to me.

But, since this is going through browserify, could you use template literals (ie, `blah blah ${value}`? Might make it a little easier to read/modify, but definitely not a requirement.

Reply:
Yup, I thought the same thing but I didn't want to add `babelify` and thought this was good enough to get rid of `eval()`

Reply:
👍 oh right, browserify doesn't transform on its own, just handles the module tree walking